python/sglang/srt/layers/attention/triton_backend.py

```diff
@@ -10,6 +10,7 @@
 from sglang.srt.model_executor.forward_batch_info import ForwardBatch
 
 if TYPE_CHECKING:
```

Comment:
add some assert to say cross attention and sliding window attention are not supported.

Reply:
https://github.com/sgl-project/sglang/blob/a1f18757eb796aff0bde9ab454e788fc970bbbd7/python/sglang/srt/model_executor/model_runner.py#L507-L519